spec/request/va_facilities_request_spec.rb

```diff
@@ -120,6 +121,36 @@
     expect(json['data'].length).to eq(12)
   end
 
+  regex_matcher = lambda { |r1, r2|
```

Comment:
These specs are a good start but don't seem comprehensive to me. I would expect to see some specs that check counts and contents of facilities/providers/combined results

Reply:
tightened the checks in these specs to make sure they're actually doing the work correctly